english/copper-plate-man/copper-plate-man.md

```diff
@@ -0,0 +1 @@
+Did you know the copper plate man can read your mind without anyone uttering a word? Remarkably, he will have answers to those questions as well without ever speaking a word. Hard to believe? [Here is his story] (http://www.jameskelleher.com/index.php/journal/copperplateman.html)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.